src/opencl_rawsha1_fmt_plug.c

```diff
@@ -192,6 +192,7 @@ static void release_clobj_kpc(void)
 		HANDLE_CLERROR(clReleaseMemObject(pinned_saved_idx), "Error Releasing pinned_saved_idx.");
 		HANDLE_CLERROR(clReleaseMemObject(pinned_int_key_loc), "Error Releasing pinned_int_key_loc.");
 		buffer_idx = 0;
+		pinned_saved_keys = 0;
```

Comment:
I'd have set them as NULL, not that it makes any difference

Reply:
Sure, let's use NULL for pointers.